src/command.ts

```diff
@@ -223,19 +226,25 @@ export class Command {
             );
           }
           process.exit();
+        })
+        .then(async (result) => {
```

Comment:
Any reason why this is here?

Reply:
Derp, no - remnant of a previous approach I tried that i didn't clean up/